phable_cli/cli.py

```diff
@@ -1,15 +1,17 @@
 import atexit
 import json
 import re
+from collections.abc import Callable
+from email.policy import default
 from pathlib import Path
-from typing import Optional
+from typing import Optional, Any
```

Comment:
```suggestion
from typing import Optional, Any, Callable
```

IIRC these are 2 different things, but I couldn't tell you why from memory. For generic types, import from `typing`.

Reply:
typing.Callable seems to be an alias to collections.abc.Callable